admin/code/LeftAndMain.php

```diff
@@ -206,7 +206,7 @@ public function getCombinedClientConfig() {
 	 */
 	public function getClientConfig() {
 		return [
-			'route' => Config::inst()->get($this->class, 'url_segment')
+			'route' => $this->Link()
```

Comment:
Just a note that this affects every admin section. Have we confirmed that the current sections (and a typical model admin) still work? That is, it's not a common practise for LeftAndMain subclasses to override `Link()` in stupid ways?